lib/vty.c

```diff
@@ -2141,6 +2150,21 @@ static void vtysh_accept(struct event *thread)
 		close(sock);
 		return;
 	}
+	/* Setting the SND buffer size to 16MB to increase the buffer size
+	 * the socket can hold before sending it to VTY shell
+	 */
+	uint32_t sndbufsize = SEND_BUF_MAX;
```

Comment:
please declare the locals at the top, follow the existing style

Reply:
Will do